### PR TITLE
Translated pots slider warnings

### LINKED ISSUE #3280: Pots & sliders preflight check menu items not translated
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### What part of EdgeTX is the focus of this bug?

Transmitter firmware

### Current Behavior

The preflight check menu items for pots & sliders can not be translated. 
They are hard coded to "OFF","ON",AUTO" for color radios and "Off","Man","Auto" for B&W radios.

### Expected Behavior

The menu items should be possible to translate and preferably also the same for color and B&W radios.

### Steps To Reproduce

Go to preflight checks under radio settings and click in the menu for pots and sliders.

### Version

2.8.0

### Transmitter

Other (Please specify below)

### Anything else?

The issue is the same for all radios.

### PR DESCRIPTION
**Pull request by ulfhedlund**
Fixes #3280 

Summary of changes:
Added string translating the preflight check menu items for pots & sliders.
String added to all translation files, set to English translation.

Translators - if you can provide the translation strings for your language I will add them to the PR.
TR_PREFLIGHT_POTSLIDER_CHECK    "Off","On","Auto"

@ParkerEde DE
@Eldenroot CZ
@robustini IT
@zyren CN / TW
@ajjjjjjjj PL
@ToshihiroMakuuchi JP
@HThuren DA

### CODE REVIEW

**Comment by HThuren** [comment]
DA
TR_PREFLIGHT_POTSLIDER_CHECK "Fra","Til","Auto"

**Comment by robustini** [comment]
In Italian ON, OFF and AUTO are already standardized, so no translation is required.

**Comment by ParkerEde** [comment]
DE
TR_PREFLIGHT_POTSLIDER_CHECK "Aus","Ein","Auto"

File: radio/src/translations/cz.h [hunk 1]
```diff
@@ -965,6 +965,7 @@
 #define TR_GLOBALVAR                   "Glob. proměnná"
 #define TR_MIXSOURCE                   "Zdroje mixeru"
 #define TR_CONSTANT                    "Konstanta"
+#define TR_PREFLIGHT_POTSLIDER_CHECK   "On","Off","Auto"
```
Comment (Eldenroot) [inline radio/src/translations/cz.h:968]: Zap, Vyp, Auto

**Review by Eldenroot — CHANGES_REQUESTED** [review]

**Comment by Eldenroot** [comment]
CZ commented.

File: radio/src/translations/cz.h (hunk 1 above)
Comment (ulfhedlund) [reply radio/src/translations/cz.h:968]: If changing the order from Off, On, Auto to On, Off, Auto would require code changes I'm not the person to do it. 
I'm not a programmer and have pretty much reached my coding limit already in this PR.

Comment (ulfhedlund) [reply radio/src/translations/cz.h]: I'll look into the order so that it is consistent in all languages, seems it isn't correct right now.

**Comment by ToshihiroMakuuchi** [comment]
JP
TR_PREFLIGHT_POTSLIDER_CHECK "OFF","ON","自動"

**Comment by ulfhedlund** [comment]
Any input from you for CN/TW and PL languages, instead of this?
#define TR_PREFLIGHT_POTSLIDER_CHECK "Off","On","Auto"

@zyren CN / TW
@ajjjjjjjj PL

**Comment by zyren** [comment]
************************************
CN

#define TR_PREFLIGHT_POTSLIDER_CHECK "关闭","开启","自动"
*****************************************
*****************************************
TW

#define TR_PREFLIGHT_POTSLIDER_CHECK "關閉","開啟","自動"
*******************************************

**Comment by ulfhedlund** [comment]
@pfeerick: I don't know what happened with the tests and why some failed after update of tw.h.

**Comment by HThuren** [comment]
> @pfeerick: I don't know what happened with the tests and why some failed after update of tw.h.

Sometimes I got sucess, when just update a single file and upload to start compilation.

**Comment by ulfhedlund** [comment]
@ajjjjjjjj
I did my best to translate to Polish using similar already existing translations.
#define TR_PREFLIGHT_POTSLIDER_CHECK "WYŁ","ZAŁ","Auto"

Pls correct if this isn't acceptable.

**Comment by pfeerick** [comment]
Thanks for this @ulfhedlund. I think Janek / aj is away/busy atm as hasn't commented on the other requests recently, so will merge this as is, and PL can be fixed up later when available again.

**Pull request merged**